Rethinking/end-of-chapter-practice-problems/ch-2.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>I think this would be even better if there were some brief explanation of how you go from line 3 to 4 - namely, how you get rid of the (C1 = 2) condition</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Good idea! I just added a causal diagram which talks about conditional independence.

Reply:
Yeah, nice add-on, it's very clear now!